.github/release.yml

```diff
@@ -0,0 +1,32 @@
+changelog:
+  categories:
+    - title: BREAKING CHANGES
+      labels:
+        - breaking
+    - title: NOTES
+      labels:
+        - note
+        - documentation
+    - title: FEATURES
+      labels:
+        - feature
```

Comment:
We also don't track features separately from enhancements (i.e. there's no such label currently) - unsure if we should and where do we even draw the line between the two

Reply:
I'm find with collapsing to Enhancements, as it can cover both an improvement or an addition while Feature means different things to different peopl

Reply:
Removed as there isn't a features label anymore